app/routes/contact/__init__.py

```
@@ -23,109 +23,190 @@
+                    
+            except Exception as e:
+                print(f"Background email operations failed: {e}")
+        
+        # Start background thread for email operations
+        email_thread = threading.Thread(target=send_emails_background)
+        email_thread.daemon = True
+        email_thread.start()
+        
+        return response_data
+        
     except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)